src/ConversionContext.h

```diff
@@ -16,7 +16,16 @@ namespace drafter {
         refract::Registry registry;
 
     public:
+        bool generateSourceMap;
```

Comment:
I understand and I thought of actually doing the same thing, but I never liked the `WrapperOptions` and I am pretty sure we will remove it soon.